### DISCUSSION TITLE
Small multiples

### DISCUSSION THREAD

**Discussion by hughess** [evidence-dev/evidence#2346] [General]
Example from Financial Times:

![image](https://user-images.githubusercontent.com/12602440/128740243-bbfdd9fa-c3b2-4a93-919d-7ccbc92830b8.png)

**Comment by PowerPlop** [comment]
@archiewood Is this already possible with an existing feature/option?

Edit (found a solution):

- select distinct list of small multiples category (e.g. Country)
- Create a loop: https://docs.evidence.dev/core-concepts/loops 
  - `{#each country_list as country}`
 - Filter your main sql query with the country parameter
 
```
<LineChart 
  data={xxxx.where(`naam = '${country.name}'`)}
  x=date
  y={['Number of deaths','xxxx']}
  series=xxxx
/>
` ``

**Reply by PowerPlop** [reply]
Maybe useful to add to the documentation?
